assembl/templates/index_react.jinja2

```diff
@@ -2,7 +2,9 @@
 {% block page_title%}Assembl{% endblock%}
 
 {% block main_css %}
-	{# Override the main css. #}
+  {# Override the main CSS. #}
+  <link type="text/css" rel="stylesheet" href="/static2/build/bundle.css" />
```

Comment:
Okay, after reading the above, it's clear that this is from the extractor used above. Is this what that is??